core/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/).
 In order to read more about upgrading and BC breaks have a look at the [UPGRADE Document](UPGRADE.md).
 
+## 1.0.22
```

Comment:
oh i just saw this, its 1.0.21 which is in progress. So please change since to 1.0.21 instead of 1.0.22